sdcm/mgmt.py

```diff
@@ -212,7 +212,7 @@ def status(self):
         Gets the task's status
         """
         cmd = "task list -c {}".format(self.cluster_id)
-        res = self.sctool.run(cmd=cmd, is_verify_errorless_result=True)
```

Comment:
why this is changed ?

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/2323/commits/adbe356dd899f75c9a687d4f26143fc94eb8b5ce